specreduce/tests/test_extract.py

```diff
@@ -14,17 +15,49 @@
 # Test image is comprised of 30 rows with 10 columns each. Row content
 # is row index itself. This makes it easy to predict what should be the
 # value extracted from a region centered at any arbitrary Y position.
-image = np.ones(shape=(30, 10))
-for j in range(image.shape[0]):
-    image[j, ::] *= j
-image = CCDData(image, unit=u.Jy)
 
 
-def test_boxcar_extraction():
+@pytest.fixture
+def mk_test_img(nrows=30, ncols=10):
+    image = np.ones(shape=(nrows, ncols))
+    for j in range(image.shape[0]):
+        image[j, ::] *= j
+    image = CCDData(image, unit=u.Jy)
+    return image
+
+
+def add_gaussian_source(image, amps=2, stddevs=2, means=None):
```

Comment:
this reminds me that i need to refactor `specreduce.synth_data.make_2d_trace_image` to have `profile` as an input parameter. right now it's hard-coded to `Moffat1D`.